static/assets/js/home.js

```diff
@@ -96,15 +108,40 @@ if (!PP) {
   };
 
   //
+  PP.showChangePasswordForm = function () {
+      $('#change_password_form')[0].reset();
```

Comment:
I would suggest `$('#change_password_form').trigger("reset");` or once we've set up Jquery.validate - `$('#change_password_form').resetForm();`